jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -6,6 +6,10 @@
 from kubernetes import client
 from jupyterhub.utils import url_path_join
 
+# Make sure that modules placed in the same directory as the jupyterhub config are added to the pythonpath
+configuration_directory = os.path.dirname(os.path.realpath(__file__))
+sys.path.insert(0, configuration_directory)
```

Comment:
I like it!

One thing I'm not sure about, @consideratio may know better: k8s may symlink config files into place, is there any situation where the real parent directories of this file and z2jh.py would be different?

Reply:
I don't know, I think though that if this works for the TravisCI test pipeline, than I think it is robust enough as it worked in that k8s setup at least.

Reply:
@manics if you mount it like
```
- mountPath: /etc/jupyterhub/z2jh.py
   subPath: z2jh.py
   name: config
```

the file is regular file, not a symlink. Symlink happens only in case of mounting secrets as a directory, which in z2jh chart is not possible to do into `/etc/jupyterhub`.
```
- mountPath: /etc/jupyterhub/config/
  name: config
```

Reply:
Anyways even if this would be moved to ConfigMap directory, the directory there is still a directory as https://docs.python.org/2/library/os.path.html#os.path.realpath ignores symlinks

```
$ root@hub-755dbdc69c-tt7r4:/srv/jupyterhub# python -c "import os;print(os.path.realpath('/etc/jupyterhub/config/z2jh.py'))"
/etc/jupyterhub/config/..2019_11_13_14_20_12.271060545/z2jh.py
```